<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="Start">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Map</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="GraphView">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit Map</string>
    </property>
   </widget>
   <widget class="QLabel" name="MapTypeLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>154</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Map Generation Method</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Choice3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>87</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Paint On Map</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Choice1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>110</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Random</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Choice2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>86</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Existing</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>148</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="NodeLabel">
     <property name="text">
      <string>Node Size    </string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QSlider" name="NodeSlider">
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>50</number>
     </property>
     <property name="singleStep">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="NodeValueLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>144</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="MapLabel">
     <property name="text">
      <string>Window Size</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QSlider" name="MapSlider">
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>500</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
